frontend/src/components/TracingIntegration/TracesFetcher.ts

```diff
@@ -72,6 +73,20 @@ export class TracesFetcher {
             'Loading traces for all clusters. Tracing is not configured to store traces per cluster.'
           );
         }
+        const firstTraceTimestamp = Math.min(...traces.map(s => s.startTime));
+        // If the traces cover less than 90% of the selected period, show an info message
```

Comment:
By the way, I just used 90% as an example, it may be a good value but if you think a different value is better then it is fine with me to change.

Reply:
Right, well, I think it is difficult to adjust. I would say between 80-90% I think

Reply:
How about 85% :-)